common/primitives/src/schema.rs

```diff
@@ -35,6 +38,28 @@ pub enum PayloadLocation {
 	Paginated,
 }
 
+/// Support for up to 16 user-enabled features on a collection.
+#[bitflags]
+#[repr(u16)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Copy, Clone, RuntimeDebug, PartialEq, Eq, Encode, Decode, MaxEncodedLen, TypeInfo)]
+pub enum Grant {
+	/// Items in this collection are transferable.
+	AppendOnly,
+	/// The metadata of this collection can be modified.
+	SignatureRequired,
```

Comment:
How is this going to work:
1. Schema with this property will ALWAYS require a signed encapsulated payload, or
2. Schema with this property may be called normally by owner or delegate, but ALSO may be used with a "one-time delegation" via a signed payload

If (2), then perhaps a different name, like `AllowSignedDelegation`, or something?

Reply:
Also, the documentation comments need to be updated.

Reply:
That will be our next story, the focus of this PR is just to update schemas pallet

Reply:
For naming

Should we be explicit about who and what is being signed?